compiler/dyno/lib/CMakeLists.txt

```diff
@@ -32,6 +32,8 @@ target_include_directories(libdyno-obj PUBLIC
 # but these are not to be public
 target_include_directories(libdyno-obj PRIVATE ${CMAKE_CURRENT_SOURCE_DIR})
 
+target_compile_options(libdyno-obj PUBLIC SHELL:$<$<COMPILE_LANGUAGE:CXX>:${DYNO_LLVM_COMP_ARGS}>)
```

Comment:
Can you add a comment about why SHELL: is needed and what the generator expression is doing?